### ISSUE TITLE
Visualization of GEO stats

### ISSUE THREAD

**Issue by raysinensis** [rnabioco/scRNA-seq-Cell-Ref-Matrix#49]
For next week's meeting, we'll again talk about the short comment article idea. 
Let's brainstorm the best way to make the code to pull data and plot as reproducible as possible.

**Comment by kriemo** [comment]
I agree. I also think we should consider looking through datasets listed in a curated list of single cell experiments and annotating if they have sufficient metadata to generate a reference. 

This preprint is a great starting point with an easy to parse google doc.
preprint:
https://www.biorxiv.org/content/10.1101/742304v2

database:
https://docs.google.com/spreadsheets/d/1En7-UV0k0laDiIfjFkdn7dggyR7jIk3WH8QgXaMOZF0/edit#gid=0

**Comment by raysinensis** [comment]
We could also start think about comparing the different data sources, see if any of them are better (don't think so...)
GEO
https://singlecell.broadinstitute.org/single_cell
https://www.ebi.ac.uk/gxa/sc/home
etc
